qiskit/pulse/configuration.py

```diff
@@ -21,6 +21,32 @@
 from .exceptions import PulseError
 
 
+class Discriminator:
+    """TODO"""
```

Comment:
These todos should be completed.

Reply:
I think you found an older version of the code